### ISSUE TITLE
PE support

### ISSUE THREAD

**Issue by shubin** [ptitSeb/box86#189]
Bonjour,
As far as I can understand, the project is primarily aimed at games and the idea is to emulate as less as possible.
I've got an idea of adding ability to run older Windows games, like Age of Empires, HOMMIII and others from that era.

From my understanding, that would require PE format loader and a set of libraries to substitute OS functionalities.
The set is going to be huge, but quite limited to file operations and DirectX functions. Speaking of AoE kind of games,  the graphics is all about DirectDraw, which can easily be emulated with SDL. 

I hope you got the idea : )
What do you think, is the idea too insane to even start working on it or it's something that can eventually work?
I think I could try, but it'd be nice to have your opinion on that and maybe some kind of advice.

**Comment by ptitSeb** [comment]
That is basically what Wine is doing in fact.

Fun fact: my initial plan, before I started working on box86, was to do that: use a ARM version of Wine and plug an x86 emulator to emulate only the core of the game and now wine and everything around. But because Wine source is quite complex and evolving rapidly, I decided to start a project from scratch... And box86 was born.

Now, doing a full alternative to Wine, even for a subset... Not sure I want to start that, it seems huge.
